### PR TITLE
[1845] Add age range to basic details

### PR DESCRIPTION
**Pull request by dankmitchell**
### Context
New age range

### Changes proposed in this pull request
Add new age range (years) to basic details

### Guidance to review
# After
localhost_3000_organisations_1JA_2019_courses_2ZRD_details(Laptop with MDPI screen)

### CODE REVIEW

**Comment by dankmitchell** [comment]
[Add age range field to basic details (MCFE)](https://trello.com/c/F2RQsTEC/1845-add-age-range-field-to-basic-details-mcfe)

File: app/views/courses/_basic_details_tab.html.erb [hunk 1]
```diff
@@ -31,6 +31,16 @@
         <dd class="govuk-summary-list__actions"></dd>
       </div>
 
+      <div class="govuk-summary-list__row">
```
Comment (fofr) [inline app/views/courses/_basic_details_tab.html.erb]: Age range shouldn't be shown for FE courses.

Comment (dankmitchell) [reply app/views/courses/_basic_details_tab.html.erb]: Extra commit with spec @fofr

File: app/views/courses/_basic_details_tab.html.erb [hunk 2]
```diff
@@ -31,6 +31,18 @@
         <dd class="govuk-summary-list__actions"></dd>
       </div>
 
+      <% unless course.level == 'further_education' %>
```
Comment (fofr) [inline app/views/courses/_basic_details_tab.html.erb]: Suggest doing this with edit options when we have them:
`if @course.meta[‘edit_options’][‘age_range_in_years’].any?`

But this is fine for now.

**Pull request merged**